core/aws.py

```diff
@@ -25,7 +25,7 @@
     )
 
 
-def upload_file(file: IO | FileStorage, bucket: str, object_name: str, metadata: dict | None = None) -> bool:
+def upload_file(file: Union[IO, FileStorage], bucket: str, object_name: str, metadata: dict | None = None) -> bool:
```

Comment:
question: `IO | FileStorage` should be the same as `Union[IO, FileStorage]` - did this need to change?